src/pages/signin/LoginForm/BaseLoginForm.js

```diff
@@ -221,7 +221,6 @@ function LoginForm(props) {
                     ref={input}
                     label={translate('loginForm.phoneOrEmail')}
                     accessibilityLabel={translate('loginForm.phoneOrEmail')}
-                    role={CONST.ACCESSIBILITY_ROLE.TEXT}
```

Comment:
@bernhardoj as you reproduced autofill issue, can you double confirm that removing this line fixes it?

Reply:
Seems like you're using autofill extension.
I am using default chrome browser autofill.

Reply:
@situchan it is the default chrome browser autofill, the manage option will redirect you to chrome://settings/autofill